features/zoek-met-bsn.feature

```diff
@@ -2,100 +2,95 @@
 
 Functionaliteit: Zoek met burgerservicenummer(s)
```

Comment:
hierin zit ook een parameter gemeenteVanInschrijving. Is het niet interessant om ook voorbeelden te hebben met de combinatie burgerservicenummer(s) en gemeenteVanInschrijving?